DigitalOffice/Services/TimeManagementService/Repositories/Filters/WorkTimeFilter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Repositories.Filters
+{
+    public class WorkTimeFilter
+    {
+        public DateTime? StartTime { get; set; }
+        public DateTime? EndTime { get; set; }
+    }
```

Comment:
It is added for the future cases, I suppose?

Reply:
Yes.